flocker/provision/test/test_digitalocean.py

```diff
@@ -0,0 +1,67 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.provision._digitalocean``.
+"""
+import copy
+import os
+
+import pyocean
+
+from twisted.trial.unittest import SynchronousTestCase, SkipTest
+
+from flocker.provision._digitalocean import (
+    set_latest_droplet_kernel, retry_if_pending)
+from flocker.testtools import random_name
+
+
+TESTING_DROPLET_ATTRIBUTES = {
+    'region': 'lon1',
+    'size': '8gb',
+    'image': 'fedora-20-x64'
+}
+
+
+class LatestDropletKernelTests(SynchronousTestCase):
+    def setUp(self):
+        """
+        Set up a test droplet and destroy it after the test.
+        """
+        token = os.environ.get('DIGITALOCEAN_TOKEN')
```

Comment:
:question: Any issue to run this on Buildbot should probably, for the implementer's sake, mention that this is necessary.